README.md

```diff
@@ -11,20 +11,19 @@ These core user-facing software include:
 - [JupyterHub](https://github.com/jupyterhub/jupyterhub)
 
 > **Important notice**
-> As of March 2019 this software **is not accessible.**
-> Significant work will be required to reach that goal.
+> As of March 2022 this software **is not accessible.**
+> Significant work will be required to reach that goal. In the meantime, you can track the progress and initiatives in our [Jupyter Accessibility Team Compass](https://jupyter-accessibility.readthedocs.io)
 
-> **Note**
-> As from June 2022 this project is an official Software Subproject as per [Jupyter Governance](https://jupyter.org/governance/software_subprojects.html)
+## Using Jupyter tools with assistive technology
 
 If you are trying to use Jupyter tools with assistive technology now, you may be able to get basic support with the following:
 
-- [Jupyter Notebook a11y toolbar](https://github.com/uclixnjupyternbaccessibility/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar) (from Microsoft research)
+- [Jupyter Notebook a11y toolbar](https://github.com/uclixnjupyternbaccessibility/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar) (from Microsoft research - ⚠️ note this has not been maintained since 2019)
 - Make cells editable with a screen reader using the [NVDA Browser Nav add-on](https://addons.nvda-project.org/addons/browsernav.en.html#:~:text=BrowserNav%20addon%20for%20NVDA,comments%20or%20malformed%20HTML%20tables.”)
 
-## :raised_hands: Getting involved
+## ✋ Getting involved
```

Comment:
Random Q: aren't emojis before the header text non-a11y friendly? I thought I remember reading somewhere that these are nuisances for people who have to read the header text from start-to-finish via audio

Reply:
I cannot remember if this is correct - maybe @isabela-pf  knows more about - in which case we can remove them or move them to the end

Reply:
I was able to find some sources to support this (there's a mention of the way it will be read in a [Perkins School for the blind blog post](https://www.perkins.org/resource/how-do-people-vision-impairments-use-emoji/)[Easter Seals blog post](https://blog.easterseals.com/emojis-and-accessibility-the-dos-and-donts-of-including-emojis-in-texts-and-emails/), [UX Collective blog post](https://uxdesign.cc/emojis-in-accessibility-how-to-use-them-properly-66b73986b803)). It seems that it's generally considered more friendly to have the words first in the case of "important" (in this case navigational) content. 

Do we want to remove the emojis or add them to the end of the header? I'm happy to make the change with whatever decision.